CMakeLists.txt

```diff
@@ -7,30 +7,29 @@
 # -DUSE_DUAL_ABI=ON (default OFF)  https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html
 # -D Boost_NO_SYSTEM_PATHS=true (default false) -D BOOST_ROOT=PATH_where_to_find_boost
 # -D AIROUTER=true (default false)
-# -D JPSFIRE=true (default false)
+# -D JPSFIRE=ON (default OFF)
 #--------------------------------------------------------------------------
-
+################################################################################
+# Project setup
+################################################################################
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 project(JPScore LANGUAGES CXX)
-
-# Project setup
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+################################################################################
+# Optional features
+################################################################################
+set(JPSFIRE OFF CACHE BOOL "Build with jps fire support")
```

Comment:
this replaces the if else block?
Then maybe it makes sense to change the  AIROUTER block as well!

Reply:
This replaces the if else yes. I did not include this since this PR was just about JPSFIRE. I have a few more cleanups for the CMake file locally ready. How about Ill include the change for the AIROUTER in an upcoming PR that just deals with CMake cleanup?

Reply:
makes sense. Could you delete this [line](https://github.com/Ozaq/jpscore/blob/get_jpsfire_compiling_again/CMakeLists.txt#L634)?

Reply:
Done, also moved the 2nd output of the JSPFIRE status up to the declaration to keep it "closer together"